.github/workflows/retest.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       # path to the retest action
       # yamllint disable-line rule:line-length
-      - uses: ceph/ceph-csi/actions/retest@28dc64dcae3cec8d11d84bdf525bda0ef757c688  # devel
+      - uses: ceph/ceph-csi/actions/retest@29aecd345fd0de42da16a000e8d4e18ee0be18df  # devel
```

Comment:
I don't think we want this. Every time there is a change merged, it will get updated?

This should just point to the files in **devel** branch. @Nikhil-Ladha do you want to make that change and make sure that the security scanner doesn't complain about it in the future?

Reply:
> I don't think we want this. Every time there is a change merged, it will get updated?

It will get updated at an interval of 1 week.

> This should just point to the files in devel branch. @Nikhil-Ladha do you want to make that change and make sure that the security scanner doesn't complain about it in the future?

I am not sure if there is a way to bypass this for security scanner, there is an open feature plan for this with no update as of now: https://github.com/orgs/ossf/projects/24/views/1?pane=issue&itemId=55431256